package.json

```diff
@@ -6,20 +6,20 @@
   "engines": {
     "node": ">=22.5.1",
     "npm": "5.6.0",
-    "yarn": "1.*"
+    "yarn": "^4.9.4"
   },
   "scripts": {
     "build:fixtures": "cp src/lib/*.json build/src/lib/",
     "build:index": "babel index.js -s inline -d build",
-    "build:lib": "babel src --copy-files --extensions '.ts,.js' --ignore src/test,src/integration,src/**/__tests__ -s inline -d build/src",
+    "build:lib": "babel src --copy-files --extensions '.ts,.js' --ignore 'src/test,src/integration,src/**/__tests__' -s inline -d build/src",
     "build:remote-schemas": "cp src/data/*.graphql build/src/data/",
     "build": "yarn build:lib && yarn build:index && yarn build:fixtures && yarn build:remote-schemas",
     "ci": "yarn test",
     "deploy-cloudflare-workers:staging": "yarn wrangler deploy --keep-vars --config=workers/caching/metaphysics-cdn.toml --env staging",
     "deploy-cloudflare-workers": "yarn wrangler deploy --keep-vars --config=workers/caching/metaphysics-cdn.toml --env production",
     "dev": "DEBUG=info,warn,error babel-node --extensions '.ts,.js' --inspect index.js",
     "dump-schema": "babel-node --extensions '.ts,.js' ./scripts/dump-schema.ts",
-    "dump:local": "yarn dump-schema _schemaV2.graphql & wait",
```

Comment:
Not sure why but after yarn upgrade this was [failing](https://app.circleci.com/pipelines/github/artsy/metaphysics/29047/workflows/fbd08572-75e6-43dc-8d74-fe60f07c48f7/jobs/59019?invite=true#step-109-337_23). Decided to just remove the `& wait`